packages/css/src/_v2-index.css

```
@@ -0,0 +1,2 @@
+@import url('./index.css');
```

Stylelint (17.11.1)

[error] Expected "url('./index.css')" to be "'./index.css'" (import-notation)

(import-notation)